### ISSUE TITLE
It doesn´t work on my system

### ISSUE THREAD

**Issue by NorlanBB** [Chem4Word/Version3#37]
After installing the full one, the Chem4word tab does not appear.

An attempt is made to force the tab to appear by clicking on "COM Add-ons" and activating the tab corresponding to the add-on.

The add-on is not activated and if you go back to the same "COM Add-ons" window it looks like it is still not activated.

- OS: Windows 11
- Word Office **365**

**Comment by MikeWilliams-UK** [comment]
1) What other add ins do you have installed?
2) Please try our latest version V3.2 https://github.com/Chem4Word/Version3-2 which is much more robust.

**Comment by MikeWilliams-UK** [comment]
This page on our web site shows that a significant proportion of people are successfully running Chem4Word on windows 11 and with Office 365.
https://www.chem4word.co.uk/usage-statistics-chemistry-addin-for-microsoft-word/

**Comment by MikeWilliams-UK** [comment]
@NorlanBB I have just examined the anonymous telemetry and it shows that 849 of 5915 distinct users are running Chem4Word with Office 365 and Windows 11

**Comment by deadlyvices** [comment]
@NorlanBB Do you have some screenshots of the problem, you can share with us? We'd very much like to solve this issue if we can.

**Comment by NorlanBB** [comment]
> 1. What other add ins do you have installed?
> 2. Please try our latest version V3.2 https://github.com/Chem4Word/Version3-2 which is much more robust.

>1. 
Acrobat PDFMaker Office COM Addin 
OneNote Linked Notes Add-in
OneNote notes about Word documents
>2.
I have tried with version 3.2 and with other versions, the problem persists


> @NorlanBB Do you have some screenshots of the problem you can share with us? We'd very much like to solve this issue if we can.

As you can see the plugin is active but the corresponding ribbon does not appear.
Also, if I go back to "COM Add-ins" it is still not activated.
![image](https://user-images.githubusercontent.com/123327659/214243083-a033bca5-985e-4462-9ca6-619c716c2067.png)
![image](https://user-images.githubusercontent.com/123327659/214243113-50e31576-0b4b-4e2f-820e-86769df0f4ea.png)
![image](https://user-images.githubusercontent.com/123327659/214243348-2c20b8e8-06c8-4576-bf78-a863ff1c6f07.png)

Microsoft 365 support indicates that your add-on has most likely been blocked

**Comment by MikeWilliams-UK** [comment]
@NorlanBB can you have a look in the folder C:\Users\[InsertUserNameHere]\AppData\Local\Chem4Word.V3\Telemetry there should be some log files which should what has gone wrong.
If you could anonymise these and post them here we will be able to look into this for you.

There is a known incompatability between Adobe Acrobat Pro which was fixed in V3.1.6

If you have been previously running V3.0.* this may have caused Chem4Word to be stuck in the list of disabled add ins.
It can be removed from this list by following the instructions here https://www.chem4word.co.uk/knowledge-base/issue-5/

**Comment by NorlanBB** [comment]
No hay nigun archivo en esta carpeta y tampoco nigun complemento deshabilitado 

There are no files in this folder and there are no disabled plugins either. (Translated to English)

![image](https://user-images.githubusercontent.com/123327659/214250733-4a0e9600-dab4-4a42-92d8-c4637f3a9af6.png)
![Captura de pantalla 2023-01-24 100340](https://user-images.githubusercontent.com/123327659/214251002-79608923-1ef3-4ee1-b190-68b78026769d.png)

**Comment by MikeWilliams-UK** [comment]
@NorlanBB Thanks for looking.

Is it possible for you to try uninstalling Acrobat Pro to see if that is still preventing Chem4Word from starting?

**Comment by NorlanBB** [comment]
> @NorlanBB Thanks for looking.
> 
> Is it possible for you to try uninstalling Acrobat Pro to see if that is still preventing Chem4Word from starting?

I've already gotten it to work, reinstalling Word completely removing registries and residual files.

I also noticed that I have confused "Disabled Items" with "Plugins slow and disabled."
The correct one looks like this. It would be nice to include a similar image in the tutorial.

![image](https://user-images.githubusercontent.com/123327659/214256074-a987107d-0dc4-49ff-8845-e34d0934a1c7.png)

On the other hand, I have already managed to activate the plugin.

![image](https://user-images.githubusercontent.com/123327659/214256269-8cec7955-8760-4a5f-86cd-2a0a4f925409.png)
